SPECS/openssh/openssh.spec

```diff
@@ -70,23 +70,16 @@ tar xf %{SOURCE1} --no-same-owner
     --with-maintype=man \
     --enable-strip=no \
     --with-kerberos5=%{_prefix}
-make
+%make_build
 
 %install
-[ %{buildroot} != "/"] && rm -rf %{buildroot}/*
-make DESTDIR=%{buildroot} install
+%make_install
 install -vdm755 %{buildroot}%{_sharedstatedir}/sshd
 
-cat <<EOF >>%{buildroot}%{_sysconfdir}/ssh/sshd_config
-AllowTcpForwarding no
-ClientAliveCountMax 2
-Compression no
-#MaxSessions 2
-TCPKeepAlive no
-AllowAgentForwarding no
-PermitRootLogin no
```

Comment:
We dropped a few of these settings with the new changes.  Are these the defaults?  And...if so...do we want to be explicit about it?  Kinda think the "PermitRootLogin no" should be maintained minimally

Reply:
Approving with suggestions.

---
In reply to: [599095906](https://github.com/microsoft/CBL-Mariner/pull/746#discussion_r599095906) [](ancestors = 599095906)

Reply:
By default PermitRootLogin enforces that root logins are only with key based authentication, no passwords. I've gone and explicitly set ```PermitRootLogin no``` so we stay consistent with the earlier config.